source/GlassView.Core/Models/BenchmarkCase.cs

```diff
@@ -0,0 +1,10 @@
+namespace GlassView.Core.Models;
+
+public sealed class BenchmarkCase : IName
+{
+    public required String Name { get; set; }
+    public required Boolean IsBaseline { get; set; }
```

Comment:
I assume you used Boolean here for consistency. Are there other reasons?

Reply:
I consistently use the CLR types (`Int32`, `String`, `Boolean`, etc.) and not the aliases (`int`, `string`, `bool`) for consistency.

If you oppose this, please let me know.

We should do a .Net session on this ;-)